document_page_environmental_aspect/__manifest__.py

```diff
@@ -20,19 +19,16 @@
 ##############################################################################
```

Comment:
You can use the new header according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1

```
# Copyright 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```